config/openshift/v4_8_exp/translate.go

```diff
@@ -121,3 +131,31 @@ func ToConfigBytes(input []byte, options common.TranslateBytesOptions) ([]byte,
 		return cutil.TranslateBytesYAML(input, &Config{}, "ToMachineConfig4_8", options)
 	}
 }
+
+func addLuksFipsOptions(mc *result.MachineConfig) translate.TranslationSet {
+	ts := translate.NewTranslationSet("yaml", "json")
+	if !util.IsTrue(mc.Spec.FIPS) {
+		return ts
+	}
+
+OUTER:
+	for i := range mc.Spec.Config.Storage.Luks {
+		luks := &mc.Spec.Config.Storage.Luks[i]
+		// Only add options if the user hasn't already specified
+		// fipsCipherOption.  Do this in-place, since config merging
+		// doesn't support conditional logic.
```

Comment:
Might be more clear to reword the first sentence to something like `Only add options if the user hasn't already specified a cipher option.` to more clearly spell out that we're bailing on any cipher being present as the constant seems like it's FIPS related unless you go look up the value.